map2.js

```diff
@@ -31,3 +31,7 @@ var phoneBook = [
 // For example, the 0th element of the new array should be {Abe: "1-111-111-1111"}
 
 //Your code here
+const newArr = phoneBook.map(number => {
+  return number
+  })
+  console.log(newArr);
```

Comment:
re-read the question; you should be changing the string of numbers in each dictionary.